content/_index.md

```diff
@@ -5,7 +5,7 @@ headline: "Jakarta<span class='tm-span'>®</span> EE"
 subtitle: "Open Source Cloud Native Java"
 description: "Jakarta Enterprise Edition (EE) is the future of cloud native Java. Jakarta EE open source software drives cloud native innovation, modernizes enterprise applications and protects investments in Java EE."
 tagline: "Powered by participation, Jakarta EE is focused on enabling community-driven collaboration and open innovation for the cloud."
-links: [[href: "about/", text: "About Jakarta EE"], [href: "https://projects.eclipse.org/projects/ee4j", text: "Projects"], [href: "specifications/", text: "Specifications"], [href: "membership/", text: "Join Us"]]
```

Comment:
Just want to make sure the intent here is to link to the Projects home page where not only Jakarta EE projects are listed.

Reply:
Nice catch!

Reply:
Yes, we are trying to push the project page initiative forward:
https://github.com/jakartaee/jakarta.ee/pull/371